py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -196,19 +191,32 @@ def set_blinding(self, is_mock, is_sv):
                                     "being ignored as mocks should not be "
                                     "blinded. 'none' blinding engaged")
                 self.blinding = "none"
-        elif is_sv:
-            if self.blinding != "none":
-                self.logger.warning(f"Selected blinding, {self.blinding} is "
-                                    "being ignored as SV data should not be "
-                                    "blinded. 'none' blinding engaged")
-                self.blinding = "none"
-        # TODO: remove this when we are ready to unblind
         else:
-            if self.blinding != "corr_yshift":
-                self.logger.warning(f"Selected blinding, {self.blinding} is "
-                                    "being ignored as data should be blinded. "
-                                    "'corr_yshift' blinding engaged")
-                self.blinding = "corr_yshift"
+            if all(self.catalogue["LASTNIGHT"] < 20210520):
```

Comment:
I just checked that the "main" survey started in May 14th 2021, so you could use here `20210514`

Reply:
changed the date